packages/common/src/constants.ts

```diff
@@ -1,14 +1,16 @@
 import { HYPHENATED_PACKAGE_VERSIONS } from './package-versions';
 
-const OFFICE_JS_LOCAL_PACKAGE_URL = `/external/office-js-${
-  HYPHENATED_PACKAGE_VERSIONS['@microsoft/office-js']
-}/office.js`;
+// For offline debugging of Office.js:
```

Comment:
It's commented out because realistically the offline debugging isn't set up yet.  But I also don't want to remove it completely, because next time I'm stuck on an airplane and don't have wifi (or next time my internet at home isn't working...), I might try to resurrect it...